.pre-commit-config.yaml

```diff
@@ -16,30 +18,36 @@ repos:
     -   id: reorder-python-imports
         files: ^src/|tests/
 
--   repo: local
+-   repo: https://github.com/pre-commit/mirrors-mypy.git
+    rev: v0.910
     hooks:
     -   id: mypy
-        name: mypy
-        entry: mypy src
-        language: system
-        pass_filenames: false
+        files: ^src/|tests/
+        # 😔 These `additional_dependencies` mirror those in the `dev` extra of `setup.cfg` and are needed
+        # to run `mypy` under `tox`. This violates "DRY" but we don't have a better way of doing this—yet.
+        additional_dependencies: [
```

Comment:
I'm not sure we need to list all these other dependencies in the template repo. I don't think every project derived from this template will need lxml, pandas, flask, jsonschema, etc… Especially not `types-waitress`, as not every Python application is going to be an HTTP server.